jelly/src/test/java/org/kohsuke/stapler/jelly/ResourceBundleTest.java

```diff
@@ -0,0 +1,63 @@
+package org.kohsuke.stapler.jelly;
+
+import hudson.util.VersionNumber;
+import io.jenkins.lib.versionnumber.JavaSpecificationVersion;
+import java.net.URL;
+import java.util.Locale;
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assume.assumeThat;
+
+public class ResourceBundleTest {
+
+    private static final URL RESOURCE_BUNDLE = ResourceBundleTest.class.getResource("/org/kohsuke/stapler/jelly/ResourceBundleTest/index.properties");
+    private static final String FILE_PATH = RESOURCE_BUNDLE.toExternalForm().replace(".properties", "");
+    private static final ResourceBundle resourceBundle = new ResourceBundle(FILE_PATH);
+
+    @Test
+    public void format() {
+        String helloWorld = resourceBundle.format(Locale.ENGLISH, "hello_world");
+
+        assertThat(helloWorld, is("Hello, World!"));
+    }
+
+    @Test
+    public void formatFallsBackToDefaultIfMissing() {
+        String helloWorld = resourceBundle.format(Locale.FRANCE, "hello_world");
+
+        assertThat(helloWorld, is("Hello, World!"));
+    }
+
+    @Test
+    public void formatLocaleOverrideInPlace() {
+        String helloWorld = resourceBundle.format(Locale.CANADA, "hello_world");
+
+        assertThat(helloWorld, is("Welcome, World!"));
+    }
+
+    @Test
+    public void formatLocaleNonISO_8859_1() {
+        String helloWorld = resourceBundle.format(Locale.CHINA, "hello_world");
+
+        assertThat(helloWorld, is("你好，世界！简化"));
```

Comment:
This is failing to compile on Windows:

```
[2022-05-06T20:55:24.497Z] [INFO] --- maven-compiler-plugin:3.10.1:testCompile (default-testCompile) @ stapler-jelly ---
[2022-05-06T20:55:24.497Z] [INFO] Changes detected - recompiling the module!
[2022-05-06T20:55:24.497Z] [WARNING] File encoding has not been set, using platform encoding Cp1252, i.e. build is platform dependent!
[2022-05-06T20:55:24.497Z] [INFO] Compiling 15 source files to C:\Jenkins\workspace\Core_stapler_PR-357\jelly\target\test-classes
[2022-05-06T20:55:24.497Z] [INFO] -------------------------------------------------------------
[2022-05-06T20:55:24.497Z] [ERROR] COMPILATION ERROR : 
[2022-05-06T20:55:24.497Z] [INFO] -------------------------------------------------------------
[2022-05-06T20:55:24.497Z] [ERROR] C:\Jenkins\workspace\Core_stapler_PR-357\jelly\src\test\java\org\kohsuke\stapler\jelly\ResourceBundleTest.java:[44,52] error: unmappable character (0x81) for encoding windows-1252
[2022-05-06T20:55:24.497Z] [ERROR] C:\Jenkins\workspace\Core_stapler_PR-357\jelly\src\test\java\org\kohsuke\stapler\jelly\ResourceBundleTest.java:[54,49] error: unmappable character (0x81) for encoding windows-1252
[2022-05-06T20:55:24.497Z] [INFO] 2 errors 
[2022-05-06T20:55:24.497Z] [INFO] -------------------------------------------------------------
```

I suspect we need to side-port [these lines](https://github.com/jenkinsci/plugin-pom/blob/fe006ce973196bd311aa53dcdaee9fccc26e221b/pom.xml#L57-L59=) from `plugin-pom` to `pom`.

Reply:
That did the trick